k8s/deploy-storageos/etcd-helpers/etcd-ansible-systemd/README.md

```diff
@@ -31,6 +31,12 @@ The playbook allows you to enable TLS with client authentication and generates
 the certificates for you. You can edit the cfssl configuration by amending the
 json files in  `roles/tls_cert/templates/`
 
+WARNING: The default CA for TLS is statically made as an example. IT SHOULD NOT BE USED FOR PRODUCTION! You need to create your own CA either with your own tools or using the script below.
+
+To create your own CA, run `sudo ansible-playbook create_ca.yaml`
```

Comment:
I'd suggest to remove the `sudo`. You can run the playbook without it. If the Docker user is configured properly in the machine, the user should be able to run docker commands.